tools/windows-node-installer/test/e2e/azure_test.go

```diff
@@ -215,6 +217,37 @@ func areRequiredRulesPresent(secGroupRules []network.SecurityRule) bool {
 	return true
 }
 
+// getIpPass extracts instance ipaddress and password.
+func getIpPass(vmName string) (string, string, error) {
+	vmCredentialPath := filepath.Join(dir, "/", vmName)
+	rdpCmd, err := ioutil.ReadFile(vmCredentialPath)
+	if err != nil {
+		return "", "", fmt.Errorf("failed to read %s credentials with error: %v", vmName, err)
+	}
+	ipAddress := ipAddressPattern.FindString(string(rdpCmd))
+	password := passwordPattern.FindString(string(rdpCmd))[3:]
+	password = strings.Trim(password, `'`)
+	return ipAddress, password, nil
+}
+
+// createHostFile creates an ansible host file and returns the path of it
+func createHostFile(ip, password string) (string, error) {
```

Comment:
If you want to do things like that, please open a separate PR to fix such things. Reordering functions and fixing formatting warrant separate PRs. Otherwise you are making the review more difficult for reviewers.